CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Fix: Multiple FlutterError.onError calls in FlutterErrorIntegration (#345)
 * Fix: Pass hint to EventProcessors (#356)
 * Fix: EventProcessors were not dropping events when returning null (#353)
+* Fix: Prefix App, Browser, Device, Message, Request and User with Sentry (#250)
```

Comment:
```suggestion
* Fix: Prefix App, Browser, Device, Message, Request and User with Sentry (#357)
```

Reply:
we're used to link the PR's id, or should we link the issue's id? wondering which one does make more sense, this suggestion is just to discuss, the Danger CI checks the PRs id btw

Reply:
Ah I see, I already wondered why it failed. 

I think both solutions are fine because PR and issue get linked. So lets keep PR IDs 👍